RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -139,6 +136,9 @@ public protocol PaywallViewControllerDelegate: AnyObject {
     @objc(paywallViewControllerWasDismissed:)
     optional func paywallViewControllerWasDismissed(_ controller: PaywallViewController)
 
+    @objc(paywallViewControlleSizeDidChange:)
+    optional func paywallViewControlleSizeDidChange(_ size: CGSize)
```

Comment:
Let's add a doc to this and the sender controller like the others have:
```suggestion
    /// For internal use only.
    @objc(paywallViewControlle:didChangeSize:)
    optional func paywallViewController(_controller: PaywallViewController, didChangeSize size: CGSize)
```